dask/sizeof.py

```diff
@@ -101,3 +101,27 @@ def sizeof_spmatrix_dok(s):
     @sizeof.register(sparse.spmatrix)
     def sizeof_spmatrix(s):
         return sum(sizeof(v) for v in s.__dict__.values())
+
+
+@sizeof.register_lazy("pyarrow")
+def register_pyarrow():
+    import pyarrow as pa
+
+    def _get_col_size(col):
+        p = 0
+        for chunk in col.data.iterchunks():
+            for buffer in chunk.buffers():
+                if buffer:
+                    p += buffer.size
+        return p
+
+    @sizeof.register(pa.lib.Table)
```

Comment:
Is pyarrow.lib public? Or should this use `pa.Table`?

same question for `pa.lib.Column` below.

Reply:
Good call - my mistake